### PR TITLE
Fix Subtle shift when hovering on "Create new file" button

### LINKED ISSUE #693: Subtle shift when hovering on "Create new file" button
nimbus-record-video-2017-09-11-22-22-33

### PR DESCRIPTION
**Pull request by CrowderSoup**
When hovering over the "Create new file" button in the repo root there
is a subtle shift in the UI due to a hidden button. This PR fixes that
issue.

Fixes #693

### CODE REVIEW

File: src/content.js [hunk 1]
```diff
@@ -532,6 +536,11 @@ function init() {
 		safely(moveAccountSwitcherToSidebar);
 	}
 
+	// Ensure that confirm buttons are always in focus
+	window.addEventListener('facebox:reveal', () => {
+		select('.facebox-content button').focus();
+	});
+
```
Comment (bfred-it) [inline src/content.js]: Unrelated change

File: src/content.js [hunk 2]
```diff
@@ -514,7 +518,7 @@ function init() {
 	//
 	// const username = getUsername();
 	// if (!username) {
-	// 	return;
+	//	return;
```
Comment (bfred-it) [inline src/content.js]: Unrelated change, will conflict with your other PR

File: src/libs/icons.js [hunk 3]
```diff
@@ -22,6 +22,6 @@ export const fork = () => <svg aria-hidden="true" class="octicon octicon-repo-fo
 
 export const cloudUpload = () => <svg aria-hidden="true" class="octicon octicon-cloud-upload" height="16" role="img" version="1.1" viewBox="0 0 16 16" width="16"><path fill-rule="evenodd" d="M7 9H5l3-3 3 3H9v5H7V9zm5-4c0-.44-.91-3-4.5-3C5.08 2 3 3.92 3 6 1.02 6 0 7.52 0 9c0 1.53 1 3 3 3h3v-1.3H3c-1.62 0-1.7-1.42-1.7-1.7 0-.17.05-1.7 1.7-1.7h1.3V6c0-1.39 1.56-2.7 3.2-2.7 2.55 0 3.13 1.55 3.2 1.8v1.2H12c.81 0 2.7.22 2.7 2.2 0 2.09-2.25 2.2-2.7 2.2h-2V12h2c2.08 0 4-1.16 4-3.5C16 6.06 14.08 5 12 5z"></path></svg>;
 
-export const darkCompare = () => <svg xmlns="http://www.w3.org/2000/svg" class="octicon octicon-diff" height="16" viewBox="0 0 13 16" width="13"><path d="M6 7h2v1H6v2H5V8H3V7h2V5h1zm-3 6h5v-1H3zM7.5 2L11 5.5V15c0 .55-.45 1-1 1H1c-.55 0-1-.45-1-1V3c0-.55.45-1 1-1zm1-2H3v1h5l4 4v8h1V4.5z" fill-rule="evenodd"/></svg>;
+export const darkCompare = () => <svg xmlns="http://www.w3.org/2000/svg" class="octicon octicon-diff" height="16" viewBox="0 0 13 16" width="15"><path d="M6 7h2v1H6v2H5V8H3V7h2V5h1zm-3 6h5v-1H3zM7.5 2L11 5.5V15c0 .55-.45 1-1 1H1c-.55 0-1-.45-1-1V3c0-.55.45-1 1-1zm1-2H3v1h5l4 4v8h1V4.5z" fill-rule="evenodd"/></svg>;
```
Comment (bfred-it) [inline src/libs/icons.js]: Unrelated change. Make sure you're workin on the latest from `master`

File: src/content.js [hunk 4]
```diff
@@ -95,6 +96,9 @@ function createMoreDropdown() {
 }
 
 function moveInsightsLink() {
+	if (select.exists('.refined-github-insights')) {
+		return;
+	}
```
Comment (bfred-it) [inline src/content.js]: Unrelated change

**Comment by bfred-it** [comment]
Can you fix the conflicts?

**Comment by CrowderSoup** [comment]
I will get them fixed this weekend! I apologize... Life got a little crazy the
past week. :)
On 2017-10-31, Federico Brigante wrote:
>Can you fix the conflicts?
>
>--
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/sindresorhus/refined-github/pull/754#issuecomment-340754235

--
Aaron Crowder, Sr. Software Dev
p: 801-903-9979
w: http://crowdersoup.com
t: https://twitter.com/crowdersoup

File: src/content.js [hunk 5]
```diff
@@ -22,6 +22,7 @@ import linkifyCode, {editTextNodes} from './libs/linkify-urls-in-code';
 import autoLoadMoreNews from './libs/auto-load-more-news';
 import addOPLabels from './libs/op-labels';
 import addReleasesTab from './libs/add-releases-tab';
+import removeUploadFilesButton from './libs/remove-upload-files-button';
```
Comment (bfred-it) [inline src/content.js]: Can you also move this file to the new `features` folder? It was created in #793 and completed by #796

**Comment by bfred-it** [comment]
Great! Can you also remove the old CSS-based version?

**Comment by bfred-it** [comment]
Thanks @CrowderSoup!

**Pull request merged**